### DISCUSSION TITLE
Color with more than 8bpp

### DISCUSSION THREAD

**Discussion by Aaron2550** [SixLabors/ImageSharp.Drawing#259] [Q&A] [answered]
Hey there. I have a Tiff Image with 48 bpp which i want to modify. However, the Color i have to give to Pens and Brushes only accepts a byte as input, no Values bigger than 255 (8bpp). How do i get a Color that is, for example, 500, 800, 100?

**Comment by JimBobSquarePants** [comment, marked as answer]
You can pass an `Rgba64` to the [`Color` constructor
](https://docs-v2.sixlabors.com/api/ImageSharp/SixLabors.ImageSharp.Color.html#SixLabors_ImageSharp_Color__ctor_SixLabors_ImageSharp_PixelFormats_Rgba64_)
